test/client_test.cc

```diff
@@ -60,7 +63,7 @@ TEST_F(ClientTest, TracingRun) {
   argv.push_back("--trace");
   argv.push_back("zipkin://localhost:9411/api/v1/spans");
   Main program(argv.size(), argv.data());
-  EXPECT_TRUE(program.run());
+  EXPECT_FALSE(program.run());
```

Comment:
That seems a dramatic change :)

Reply:
Yes .. also see https://github.com/envoyproxy/nighthawk/pull/167/files/bcc8e4fee772dbbc967d976145e2a9fbdd3465d0#r337765347
I have added comments about this (also in this file), fortunately the python integration tests cover this with a happier flow (a running server to talk to).